static/css/foundation.min.css

```diff
@@ -619,11 +619,11 @@ h6, .h6 {
 
 a {
   line-height: inherit;
-  color: #1779ba;
```

Comment:
@davidprojectsbyif Can we make these changes in main.css? If Foundation updates their CSS file, that would wipe any changes we've made to it. 

At the top of main.css, around where the h1, h2, h3 rules are, you can put something like: 
`a {
   color: #242249;
}

a:hover, a:focus {
  color: #010e7e;
}
`

And change the foundation.min.css back to defaults. This should have the same changes, but in a file that we can control